tools/serverpod_cli/lib/src/test_util/builders/class_definition_builder.dart

```diff
@@ -22,6 +22,8 @@ class ClassDefinitionBuilder {
   List<_FieldBuilder> _fields;
   List<SerializableModelIndexDefinition> _indexes;
   List<String>? _documentation;
+  InheritanceDefinition? _subClass;
+  InheritanceDefinition? _baseClass;
```

Comment:
Suggestion: _subClass should be a list, and remember to rename these if you rename `base` to `super`